src/adapters/scene_direction.js

```diff
@@ -93,7 +93,7 @@ export default class SceneDirection {
       }
 
       // Custom markers
-      if (this.vehicle !== "walking" && !Device.isMobile()) {
+      if (!Device.isMobile()) {
```

Comment:
not in the scope of this bug, but yes. when we ask mapbox to draw equally spaced symbols  along a path, it will avoid the steps coordinates, so they should conserve their step marker instead of showing a hole